packages/fxa-geodb/test/maxmind-db-downloader.js

```diff
@@ -47,20 +47,6 @@ describe('maxmind-db-downloader', function () {
         'Directory was created'
       );
     });
-
-    it('does nothing when directory already exists', function () {
```

Comment:
The test above does the same thing pretty much.

Reply:
I noticed this test breaks now. I wonder if the test should just be updated instead of removed since it does seem like an valid edge case to me. That being said, given the example usage in the readme, it doesn't really seem to matter.